src/components/Legend.js

```diff
@@ -22,9 +22,23 @@ export default class Legend {
       .attr('x2', '100%')
       .attr('y2', '0%');
 
-    legendGradient.append('stop')
-    .attr('offset', '0%')
-    .attr('style', `stop-color:${this.map.options.countries.attr.fill.min};stop-opacity:1`);
+    if (this.map.options.countries.attr.fill.minNegative &&
+        this.map.options.countries.attr.fill.maxNegative) {
+      legendGradient.append('stop')
+          .attr('offset', '0%')
+          .attr('style', `stop-color:${this.map.options.countries.attr.fill.maxNegative};stop-opacity:1`);
+
+      legendGradient.append('stop')
+          .attr('offset', '49%')
```

Comment:
yes and no ;)  yes, I agree that my new code should check to see what the smallest and largest numbers are and put the percentages in accordingly (*not* covered by d6a98e8, commit pending).  However, no, the problem of negative numbers flying off the legend exists in master per #27.  So i'll do the work to adjust what % we use for these stops, but it won't fix the legend.